### PR TITLE
Allowed Login Documentation

### PR DESCRIPTION
**Pull request by russjones**
**Purpose**

Updated Trusted Clusters documentation to add details on how to update `allowed_logins`.

**Implementation**

n/a

### CODE REVIEW

**Comment by klizhentas** [comment]
retest this please

**Comment by klizhentas** [comment]
retest this please

**Comment by russjones** [comment]
@gravitational-jenkins test this please

File: docs/2.0/admin-guide.md [hunk 1]
```diff
@@ -826,6 +826,89 @@ Similarly, by passing `--cluster=cluster-b` to `tsh` John can login into cluster
     Teleport Enterprise also supports adding and removing trusted clusters dynamically
     at runtime. See [this section](enterprise.md#dynamic-trusted-clusters) to learn more.
 
+
+### Updating Allowed Logins
```
Comment (klizhentas) [inline docs/2.0/admin-guide.md]: I would change the title to "Permissions with Trusted Clusters"

**Pull request merged**